anitya/db/models.py

```diff
@@ -339,6 +339,9 @@ class Project(Base):
     latest_version = sa.Column(sa.String(50))
     logs = sa.Column(sa.Text)
 
+    last_check = sa.Column(sa.DateTime, default=datetime.datetime.utcnow)
+    next_check = sa.Column(sa.DateTime, default=datetime.datetime.utcnow)
```

Comment:
Oh, I forgot to mention this on my first review, but you're querying on this column so it should probably be indexed.